frame/dapps-staking/src/lib.rs

```diff
@@ -83,10 +81,7 @@ pub type BalanceOf<T> =
 pub type EraIndex = u32;
 
 /// Simple trait used to check whether the underlying struct represents a valid smart contract on-chain.
```

Comment:
Docs should also be updated, now it's just a wrapper.

Perhaps the trait bounds from the pallet config could just be put here?
Especially the part about encoding.

Reply:
I tried to inherit `Parameter` and `Member` for `IsContract` trait, but this is not possible. `frame_support::traits::dispatch::{Parameter, Member}`
`dispatch` is private module and not accessible directly. I think `Parameter` and `Member` are available in `#[pallet::config]`